### PR TITLE
fix issue 41

### PR DESCRIPTION
**Pull request by nguyenzung**
It seems better if adding other variable on L2 instead of increasing RELAY_GAS_CHECK_BUFFER for both L1, L2. 
We don't need to increase gas for L1CrossDomainMessenger.sendMessage or  L1CrossDomainMessenger.sendNativeTokenMessage ( will be relayed by L2CrossDomainMessenger.relayMessage), it would be a waste because of burning step. https://github.com/tokamak-network/tokamak-thanos/blob/main/packages/tokamak/contracts-bedrock/src/L1/OptimismPortal2.sol#L524

Thank you!

### CODE REVIEW

File: op-bindings/bindings/faultdisputegame.go
Comment (0x6e616d) [inline op-bindings/bindings/faultdisputegame.go:1]: Ignore these changes

File: op-bindings/bindings/uniswapv3factory_more.go
Comment (0x6e616d) [inline op-bindings/bindings/uniswapv3factory_more.go:1]: Ignore these changes

**Review by 0x6e616d — COMMENTED** [review]
Thank you, please take a look at [this line](https://github.com/tokamak-network/tokamak-thanos/blob/main/packages/tokamak/contracts-bedrock/src/L1/L1CrossDomainMessenger.sol#L281
). We should use the new BUFFER constant for checking the minGas

**Comment by nguyenzung** [comment]
> Thank you, please take a look at [this line](https://github.com/tokamak-network/tokamak-thanos/blob/main/packages/tokamak/contracts-bedrock/src/L1/L1CrossDomainMessenger.sol#L281). We should use the new BUFFER constant for checking the minGas

Thank you!
I updated!

**Review by rlgns98kr — APPROVED** [review]
Thank you very much, Brave!

**Review by theo-learner — APPROVED** [review]
+1
Thank you so much!

**Comment by theo-learner** [comment]
I created the new base branch for submitting fixed PR. Could you change the base branch to fix-review(https://github.com/tokamak-network/tokamak-thanos/tree/fix-review)?

**Comment by nguyenzung** [comment]
> I created the new base branch for submitting fixed PR. Could you change the base branch to fix-review(https://github.com/tokamak-network/tokamak-thanos/tree/fix-review)?

It's done!

**Pull request merged**